client/src/app/utils/waves-selector.ts

```diff
@@ -34,17 +34,12 @@ export const getWavesWithStatus = (
     const statuses = getTicketStatus(wave, tickets);
     if (statuses.includes("Error")) {
       return "Error";
-    } else if (statuses.includes("")) {
-      const now = dayjs.utc();
```

Comment:
Are we not handling the starting date anymore for the "In progress" aggregate status value ?

Reply:
I was thinking of this and wondering where the start date logic falls in with the other status we are deriving from JIRA. When would we prioritize using the date to derive status over using the status of the issues to derive status? Seems as if there should be a state for when all issues are in "New" or "Not started" state but the migration date is technically "In progress", for example.